Makefile

```
@@ -0,0 +1,57 @@
+PROJECT := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+SRC := $(PROJECT)src
+TESTS := $(PROJECT)tests
+ALL := $(SRC) $(TESTS)
+
+export PYTHONPATH = $(PROJECT):$(PROJECT)/lib:$(SRC)
+
+update-dependencies:
+	uv lock -U --no-cache
+
+generate-requirements:
```

If you adopt [this suggestion](https://github.com/canonical/tempo-coordinator-k8s-operator/pull/62/files#r1842549141), then:
```suggestion
generate-requirements: clean-requirements
```

I do this because the real `pip-compile` is a bit unpredictable when a requirements.txt already exists, and I don't know how closely `uv pip compile` models the original's behavior.  This feels safer